contracts/voting_escrow/src/contract.rs

```diff
@@ -565,6 +624,10 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
                 deposit_token_addr: config.deposit_token_addr.to_string(),
             })
         }
+        QueryMsg::Balance { address } => to_binary(&get_user_balance(deps, env, address)?),
+        QueryMsg::TokenInfo {} => to_binary(&query_token_info(deps)?),
```

Comment:
total_supply in TokenInfo's response is always 0. It's better to return total voting power as a total supply in response.

Reply:
Yep, thanks. Already fixed